packages/cli/.prettierrc

```diff
@@ -1,6 +0,0 @@
-{
-  "bracketSpacing": false,
```

Comment:
What is the reason for removing `.prettierrc` from this project? I am not against such change per se, but would like to better understand the reasoning. It may be better to make this change in a standalone pull request.

Reply:
The reason was that this `.prettierrc` uses different formatting that the top-level one we use for the project and I noticed that and removed it. I've reverted this change and will make a new PR for this.